### PR TITLE
Usability updates

### PR DESCRIPTION
**Pull request by hectorsector**
This PR tries to improve the flow of this course by applying some changes to its usability.

### CODE REVIEW

**Comment by hectorsector** [comment]
👀 @crichID working through some of the changes we talked about to this course, should be ready to ship soon!

**Comment by hectorsector** [comment]
@a-a-ron @brianamarie @mjcastner @mattdavis0351 @lmkeston @crichID could you please 👀 this PR? I can merge with one or two approvals.

**Comment by a-a-ron** [comment]
Nice work @hectorsector. I think the suggested changes will help the learner progress more easily through the course. I did notice a few minor things.

- The `Create a README` step asks the learner to update two different placeholders in the file. With these two suggested changes, if the learner accepts the first change, they get the "error" response that really isn't an error but a response to make the other change. The learner then accepts the other suggested change to move forward. Although these suggested changes are direct to the lines being changed, it may be best to use one multiline suggested change that the user can accept instead of seeing another prompt response to make the other change. 

This may make the suggest change look quite big so 🤷‍♂. Just a thought. 

- The step to add a label is 8 steps long. It may be good to include a gif to highlight the steps to make it easier to grasp the steps all at once. A learner may be scrolling back and forth to read the instructions

File: responses/06-add-labels.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ These initial labels are awesome, but most maintainers add (or delete) labels to
 
 1. On the top right side of this pull request view, find the section titled: "Labels".
 1. Click the gear icon.
-1. Notice the text field at the top says **Filter or create labels**.
+1. Notice the text field at the top says **Filter labels**.
```
Comment (mattdavis0351) [inline responses/06-add-labels.md]: That text field no longer says "filter or create". Just "filter labels"

Comment (mattdavis0351) [reply responses/06-add-labels.md]: image

**Comment by mattdavis0351** [comment]
@hectorsector I had no hang-ups.  Just a difference in what the UI was showing the user when it came to labels

**Pull request merged**